### PR TITLE
WIP BXMSDOC-6898: Added accept-language as HTTP Header for REST API

### PR DESCRIPTION
**Pull request by hmanwani-rh**
See JIRA: https://issues.redhat.com/browse/BXMSDOC-6898

Doc previews:
**Interacting with Red Hat Process Automation Manager using KIE APIs**
[28.1. Sending requests with the Knowledge Store REST API using a REST client or curl utility](http://file.pnq.redhat.com/~hmanwani/BXMSDOC-6898-RHPAM/#knowledge-store-rest-api-requests-proc_kie-apis)

**Interacting with Red Hat Decision Manager using KIE APIs**
[28.1. Sending requests with the Knowledge Store REST API using a REST client or curl utility](http://file.pnq.redhat.com/~hmanwani/BXMSDOC-6898-RHDM/#knowledge-store-rest-api-requests-proc_kie-apis)

### CODE REVIEW

File: doc-content/shared-kie-docs/src/main/asciidoc/Workbench/RemoteAPI/knowledge-store-rest-api-requests-proc.adoc [hunk 1]
```diff
@@ -141,19 +142,20 @@ For curl utility:
 * `-u`: Enter the user name and password of the {CENTRAL} user with the `rest-all` role.
 * `-H`: Set the following headers:
 ** `Accept`: `application/json`
+** `Accept-Language`: `en-US`
```
Comment (akumar074) [inline doc-content/shared-kie-docs/src/main/asciidoc/Workbench/RemoteAPI/knowledge-store-rest-api-requests-proc.adoc]: Can we add the default case info here? wdyt?

**Review by akumar074 — CHANGES_REQUESTED** [review]

File: doc-content/shared-kie-docs/src/main/asciidoc/Workbench/RemoteAPI/knowledge-store-rest-api-requests-proc.adoc (hunk 1 above)
Comment (hmanwani-rh) [reply doc-content/shared-kie-docs/src/main/asciidoc/Workbench/RemoteAPI/knowledge-store-rest-api-requests-proc.adoc]: Done, have added a note.

**Review by akumar074 — APPROVED** [review]
LGTM! Thanks for the PR.

**Review by kaldesai — APPROVED** [review]
Hey @hmanwani-rh , your accept-language HTTP Header changes looks good to me. Thank you.

**Review by barboras7 — APPROVED** [review]
Looks good, thanks Heena

**Pull request merged**